docs/prow/prow-k8s-testgrid.md

```diff
@@ -1,73 +1,63 @@
 # TestGrid
 
-Kyma publishes Prow job results on the [**Kubernetes TestGrid**](https://testgrid.k8s.io/kyma-all) that the Kubernetes team runs. Prow job configuration for Kyma is stored in two separate projects, in Kyma and Kubernetes `test-infra` repositories.
+[TestGrid](https://testgrid.k8s.io) is an interactive dashboard for viewing tests results in a grid. It parses JUnit reports for generating grid view from the tests.
+There is one dashboard group called `kyma` which groups dashboards from both of the organizations: *kyma-project* and *kyma-incubator*. The dashboards start with the prefix name which refers to one of the organizations: *kyma* and *kyma-incubator*.
+TestGrid configuration is stored inside [kubernetes/test-infra](https://github.com/kubernetes/test-infra/tree/master/config/testgrids/kyma) repository in the `/config/testgrids/kyma/` directory.
 
-## Kyma configuration
+TestGrid is being automatically updated by Prow using the tool called [transfigure.sh](https://github.com/kubernetes/test-infra/tree/master/testgrid/cmd/transfigure).
 
-One part of the configuration is based on job definitions in the form of annotations. See the example of the [`kind` job](https://github.com/kyma-project/test-infra/blob/60493dd61d77da363b8758b7e4c94f25d4b36501/prow/jobs/test-infra/test-infra-kind.yaml#L80-L83) displayed on the *kyma-nightly* dashboard on TestGrid:
+## Adding new dashboard
 
+The dashboards' configuration is stored in the [testgrid-default.yaml](https://github.com/kyma-project/test-infra/tree/master/prow/testgrid-default.yaml) file.
+This file is being automatically generated from a [template testgrid-default.yaml](https://github.com/kyma-project/test-infra/blob/master/templates/templates/testgrid-default.yaml) file.
 ```yaml
-  annotations:
-    testgrid-dashboards: kyma-nightly
-    # testgrid-alert-email: email-here@sap.com
-    testgrid-num-failures-to-alert: '1'
+dashboards:
+  # kyma
+  - name: kyma_integration
+  - name: kyma_control-plane
+
+  # kyma-incubator
+  - name: kyma-incubator_compass
+
+dashboard_groups:
+  - name: kyma
+    dashboard_names:
+      - kyma_integration
+      - kyma_control-plane
+      - kyma-incubator_compass
 ```
+To add a new dashboard it's needed to do several steps:
 
-You can also specify an email address if you want to receive notifications after a predefined number of job failures.
+1. Add a new dashboard definition in the `dashboards` field according to the example file above inside a template file. **Dashboards need to have dashboard name as prefix**.
```

Comment:
```suggestion
1. Add a new dashboard definition in the `dashboards` field inside a template file. **Dashboards must have a dashboard name as a prefix**.
```

Reply:
Dashboard definition or configuration? 🤔

Reply:
It is a name of a dashboard that has to be used in the prowjob annotations. I may be *dashboard name* as well.